### ISSUE TITLE
Accessibility: dragging chips with screen readers is difficult or impossible

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#1126]
### Bug Description

In a recent version, the ability to customize the appearance of your cards and posts was added with the ability to drag the items you wanted on or off. However, screen readers like TalkBack and VoiceOver do not do well with drag actions like that as they are very visual.

### Expected Behaviour

Screen reader users should be able to perform this action as well. My suggestion is make these items tapable to move them from one area to the other.

### Steps to Reproduce

Turn on TalkBack from Settings, Accessibility, TalkBack
Open Thunder
Tap settings, appearance, posts
Try dragging the items by single tapping on one to highlight it, and then double tap and hold, and then drag, and you will see the issue as this is very difficult.

### Additional Context

![Screenshot_20240213-133002_Thunder](https://github.com/thunder-app/thunder/assets/116814522/0baf7485-e60f-46d3-abef-ca4f24b54a5c)

### App Version

0.2.9-5

### Device

Oneplus Nord N200

### OS

Android 13
